src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,7 +66,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
         $loader->load('guard.xml');
 
-        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
+        if ($container->has('profiler')) {
```

Comment:
Sure. I didn't use `$container->has('profiler')` at first. I used `$container->getParameter('kernel.debug')` but I was told that it was not completely right. But I still don't understand, so I can't fix it.